### DISCUSSION TITLE
Change `Battery` under `Devices` to `Power` and show devices that are running on mains

### DISCUSSION THREAD

**Discussion by mehuman** [home-assistant/frontend#22219] [Configuration]
Currently under the `Devices` dashboard devices with a battery show the percentage of battery left and devices running on mains power just show `-`. It would be great if those on mains showed a plug icon.

My use case is looking through devices it can be difficult to tell if a device has fully dropped or is just running on mains.

**Comment by karwosts** [comment]
Do most of your devices themselves generally have a consistent way to report that they are running on mains power?

Generally we detect battery by looking for an entity (sensor or binary_sensor) of a device that has device_class: battery. 

How would we detect mains powered devices? There is a binary_sensor with device_class power, but I don't know if it is consistently used for this. E.g. I have some battery devices that also have a binary power sensor. (Xiaomi temperature/humidity sensors). 

I think it's a decent idea but I don't know if we have enough information to reliably implement it.

**Comment by mehuman** [comment]
Including two screenshots of zigbee devices that present a power source of `Mains`. The dashboard could just show the plug icon when the power source `Mains` is detected in the same way when power source is `Battery`. If neither detected it could revert back to the `-`.

![Screenshot from 2024-10-07 10-41-27](https://github.com/user-attachments/assets/eb15dd3e-0d66-414b-9dc5-d51c8b3a2b28)
![Screenshot from 2024-10-07 10-40-54](https://github.com/user-attachments/assets/cd04b68e-e16d-4430-be4a-feb882a5a7db)
